report/runTest.ecl

```diff
@@ -0,0 +1,9 @@
+IMPORT ^ as DataPatterns;
+
+filePath := 'heart_disease_uci.csv';
+
+ds := DATASET(filePath, RECORDOF(filePath, LOOKUP), CSV);
+
+profileResults := DataPatterns.Profile(ds);
+
+OUTPUT(profileResults, ALL, NAMED('profileResults'));
```

Comment:
Will probably remove from release version?

Reply:
Maybe it could be modified to use a small-ish inline dataset as a sort of stand-alone "hello world" example of DataPatterns?